build/build.bat

```diff
@@ -162,6 +165,42 @@ set TARGET_VS_CONFIG="%TARGET_PROJ_CONFIG% %BUILD_FLAVOR%|%BUILD_ARCH%"
 pushd %BUILD_DIR%
 call "%~dp0\SetVsEnv.bat" %TARGET_PROJ_CONFIG%
 
+rem Check for x86 viosock libraries and build them if needed...
+if "%BUILD_FILE%"=="virtio-win.sln" (
```

Comment:
Can he have this block as a separate batch file called form build.bat (prebuild_x86_vsock.bat)?
In the future, we will most probably have something similar for virtio-gpu as well.

Reply:
We can, but it's only worthwhile if it's the whole thing, less some error checking on return.
This will suit inclusions for other drivers too, e.g. your mention of `virtio-gpu`.
The only downside to this is that it will always be called - for every build - until retired.

Reply:
Okay, so I've dropped that into `prebuild_x86_libs.bat`.

Reply:
Just one minor cleanup... 8^d

Reply:
Ok, all sorted...